supergrub-mkcommon

```diff
@@ -66,21 +66,52 @@ function generate_and_copy_mo_files () {
 }
 
 function check_unifont () {
+  local unifont_basename="unifont.pf2"
```

Comment:
As I understand from Ubuntu 22.04 this is now `unicode.pf2` and we should be using that instead.

Reply:
If it's easy for you to fix, please do that; I don't have a local repo set up currently.

Reply:
Nevermind, found one.